bepdf/beos/Annotation.cpp

```diff
@@ -589,6 +589,36 @@ void TextAnnot::Print() {
 	fprintf(stderr, "Open: %s\n", IsOpen() ? "true" : "false");
 }
 
+LinkAnnot::LinkAnnot(LinkAnnot* copy)
+	: Annotation(copy),
+	mLinkAction(NULL)
+{
+}
+
+
+LinkAnnot::LinkAnnot(Dict* d)
+	: Annotation(d),
+	mLinkAction(NULL)
+{
+	Object obj;
+
+	// look for destination
+  if (d->lookup("Dest", &obj) && !obj.isNull()) {
+    mLinkAction = LinkAction::parseDest(&obj);
+  // look for action
+  } else {
+    if (d->lookup("A", &obj) && obj.isDict()) {
+      mLinkAction = LinkAction::parseAction(&obj/*, baseURI*/);
+    }
+  }
+  obj.free();
```

Comment:
The rest of the file is in tabs, but this is in spaces. Koder bug? :p

Reply:
1. I do not see .editorconfig in this repo. Where would the bug be? :)
2. I was more occupied reverting all the removed whitespace to actually notice that indentation is inconsistent. ;)

Reply:
Oh, can't Koder auto-detect the use of tabs in most source files? I thought it did...

Reply:
It overrides settings only if there is .editorconfig in the repo (and that has a few bugs). There is no auto-detection.